app/views/content/landing/careers/_mailing_list.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <section class="col col-720">
+    <%= render Content::MailingListComponent.new(
+      title: "Discover how exciting teaching can be",
+      intro: "Get one step closer to the classroom and find out what teaching's really like. If you're aged 16 years or over, sign up for our mailing list and get inspiring stories from people training to teach, and early career teachers, sent straight to your inbox. Just answer a few quick questions about yourself."
```

Comment:
we avoid saying 'mailing list' externally - could we just say something like 'sign up to get....'?

Reply:
Is talking about ECTs too much detail for this audience? Worth just saying something like 'teachers and trainees' to keep it simple?

Also - worth checking with DM that this is accurate!

Reply:
Also - should we add `advice` in here - it is not just stories?

Reply:
@gemmadallmandfe I only put stories as at this point would they be looking for advice on how to get into teaching as it's quite far off that they would apply? Maybe a question for pipeline/ DM team?

And I wasn't sure what info they would get sent as we don't have anything specific for this audience. I thought case studies may be the most relevant to mention to inspire them. But again I can send this to the DM team and ask what they feel should be included here.